plugin/trino-hive/src/main/java/io/trino/plugin/hive/s3select/S3SelectCsvRecordReader.java

```diff
@@ -91,4 +92,11 @@ public SelectObjectContentRequest buildSelectObjectRequest(Properties schema, St
 
         return selectObjectRequest;
     }
+
+    @Override
+    protected String getFieldDelimiter(Properties schema)
+    {
+        // Use the field delimiter only if it is specified in the schema. If not, use null (defaults to ',' in S3 Select).
+        return schema.getProperty(FIELD_DELIM);
```

Comment:
It sounds like we should have S3 Select tests for csv files with various delimiters, not only the default one.